.github/workflows/test-code-code2parquet.yml

```diff
@@ -0,0 +1,95 @@
+#
+# DO NOT EDIT THIS: it is generated from test-transform.template,  Edit there and run make to change these files
+#
+name: Test - transforms/code/code2parquet
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - "dev"
+            - "releases/**"
+        tags:
+            - "*"
+        paths:
+            - "transforms/code/code2parquet/**"
```

Comment:
not sure about that... in a release the image of this transform will not be pushed if there was no change in it?

Reply:
i suppose this begs the need for a separate workflow for publishing and to not include publishing with testing.